### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by AilinSoon**

### CODE REVIEW

**Comment by AilinSoon** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4', '>= 6.1.4.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (huongnt-2545) [inline Gemfile]: using `gem 'mysql2'` instead of

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MalayGroup2</title>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb:4]: I18n

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (huongnt-2545) [inline config/database.yml]: Please follow comments by trainers for the previous tutorial
- gitignore this file
- create new file is named `database.yml.example` in `config/` folder
- config db use MySQL (refs: https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database)

**Review by huongnt-2545 — COMMENTED** [review]
I only comment the init project pull request for 1 member in your team

**Comment by AilinSoon** [comment]
ready

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# MySQL. Versions 5.5.8 and up are supported.
+#
+# Install the MySQL driver
+#   gem install mysql2
+#
+# Ensure the MySQL gem is defined in your Gemfile
+#   gem 'mysql2'
```
Comment (ngocvt-0484) [inline config/database.yml]: please 
```
git -rm --cached  config/database.yml
git add .
git commit --amend --no-edit
git push origin name_branch -f
```

**Comment by AilinSoon** [comment]
ready

**Comment by ngocvt-0484** [comment]
not fix yet. dont push config/database.yml to github

**Comment by AilinSoon** [comment]
ready

**Comment by AilinSoon** [comment]
ready

**Comment by AilinSoon** [comment]
ready

**Comment by AilinSoon** [comment]
ready

**Pull request merged**